src/Simplify_Call.cpp

```diff
@@ -74,6 +74,24 @@ int ctz64(uint64_t x) {
 #endif
 }
 
+// Rewrite name(broadcast(args)) to broadcast(name(args)).
+// Assumes that scalars are implicitly broadcasted.
+Expr lift_elementwise_broadcasts(Type type, const std::string &name, std::vector<Expr> args, Call::CallType call_type) {
+    if (type.lanes() == 1) {
+        return Expr();
+    }
+    for (Expr &i : args) {
+        const Broadcast *b = i.as<Broadcast>();
+        if (b) {
```

Comment:
nit: `if (const Broadcast *b = i.as<Broadcast>())` would be terser and idiomatic.